release-announcement.adoc

```diff
@@ -98,3 +98,52 @@ class MyEmbeddable { ... }
 
 ----
 
+[[embeddable-inheritance]]
+== Embeddable Inheritance
+
+Another new feature of this version is discriminator-based inheritance for `@Embeddable` types. An `@Embeddable` class
+may be extended by other `@Embeddable` classes, in which case the `@Embedded` properties using that type will
+rely on an additional discriminator column to store information about the composite value's subtype.
+
+When retrieving the inherited embedded property, Hibernate will read the discriminator value and instantiate the
+correct `@Embeddable` subtype with its corresponding properties.
+
+For example, a mapping like this:
+[source,java]
+----
+@Embeddable
+@DiscriminatorValue( "parent" )
+@DiscriminatorColumn( name = "embeddable_type" )
```

Comment:
I guess `@DiscriminatorFormula` is also supported? Maybe that's worth mentioning?

Reply:
Sure, I'll mention it in the user guide.